sessionctx/variable/sysvar.go

```diff
@@ -69,7 +69,7 @@ const (
 var (
 	UnknownStatusVar   = terror.ClassVariable.New(CodeUnknownStatusVar, "unknown status variable")
 	UnknownSystemVar   = terror.ClassVariable.New(CodeUnknownSystemVar, "unknown system variable '%s'")
-	ErrIncorrectScope  = terror.ClassVariable.New(CodeIncorrectScope, "Incorrect variable scope")
+	ErrIncorrectScope  = terror.ClassVariable.New(CodeIncorrectScope, "Variable '%s' is a %s variable")
```

Comment:
Why not use https://github.com/pingcap/tidb/blob/3db7594fa53c9e2dfb71f186231d3723a817b384/mysql/errname.go#L256 ?

Reply:
Indeed. These error codes can be refined. Will fix